src/scirpy/pl/_logoplots.py

```diff
@@ -0,0 +1,89 @@
+from collections.abc import Sequence
+from typing import Literal
+
+from logomaker import Logo, alignment_to_matrix
+
+from scirpy.get import airr as get_airr
+from scirpy.util import DataHandler
+
+
+@DataHandler.inject_param_docs()
+def logoplot_cdr3_motif(
+    adata: DataHandler.TYPE,
+    chains: Literal["VJ_1", "VDJ_1", "VJ_2", "VDJ_2"] | Sequence[Literal["VJ_1", "VDJ_1", "VJ_2", "VDJ_2"]] = "VDJ_1",
+    airr_mod="airr",
+    airr_key="airr",
+    chain_idx_key="chain_indices",
+    cdr3_col: str = "junction_aa",
+    to_type: Sequence[Literal["information", "counts", "probability", "weight"]] = "information",
+    pseudocount: float = 0,
+    background=None,
+    center_weights: bool = False,
+    plot_default=True,
+    **kwargs,
+):
+    """
+    A user friendly wrapper function for the logomaker python package.
```

Comment:
```suggestion
    Make Logoplots of CDR3 sequences.

    This is a wrapper around the logomaker package.
```

First line should always be a concise summary of what a function does, followed by a blank line.